hackathon_site/dashboard/frontend/src/pages/Dashboard/Dashboard.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Dashboard from "./Dashboard";
+
+it("renders correctly when the dashboard appears with the sponsor card", () => {
+    const { asFragment } = render(<Dashboard />);
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
If you look at the snapshot for this test, it's massive because it's also rendering everything within this component (testing-library doesn't have shallow rendering). 

A better test (that's more inline with RTL's guiding principles) would be to look for the text you expect to be in eg the title of each card and make sure it's there.